mlir/include/mlir/Conversion/Passes.td

```diff
@@ -40,7 +40,15 @@ def ConvertToSPIRVPass : Pass<"convert-to-spirv"> {
   let description = [{
     This is a generic pass to convert to SPIR-V.
   }];
-  let dependentDialects = ["spirv::SPIRVDialect"];
+  let dependentDialects = [
+    "spirv::SPIRVDialect",
+    "vector::VectorDialect",
+  ];
+  let options = [
+    Option<"runSignatureConversion", "run-signature-conversion", "bool",
+    /*default=*/"false",
+    "Run function signature conversion to convert vector types">
+  ];
```

Comment:
Why not always run it?

Reply:
The `GreedyPatternRewriteDriver` performs dead-code elimination, which will affect the other `ConvertToSPIRV` unit tests (e.g. functions with no return value and just check for individual dialect conversions).

Reply:
Ah, I see. Then if updating these tests to return the values is too much work, I'd instead make the signature conversion run by default and make those tests disable it.